src/ui/components/Icon/styles.scss

```diff
@@ -40,6 +40,16 @@
   width: 112px;
 }
 
+.Icon-triangle-down {
+  background: url("./triangle-down.svg") center no-repeat;
```

Comment:
Wait, sorry, I just realised exactly what you were asking. Yeah this PR started awhile back so I was missing it I think. I'll fix that!